src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -59,6 +72,10 @@ export interface NavBarScheduleProps {
   venueId: string;
 }
 
+const minRangeValue = 0;
+const todaysDate = new Date();
+const todaysDateTime = new Date().getTime();
```

Comment:
ideally, these constants should not be stored this way. Actually, todasDate can change if a user enters the platform at 23:58 and stays till the next day - todaysDate and the schedule should be updated.

let's mark it as OOS of the PR and will fix that in the follow-up PR, because otherwise it feels like we'll never merge this one.. 🙈

Reply:
removed `todaysDateTime` completely and will remove `todaysDate` in follow up PR 58b5cf2